cognite/experimental/_api/alerts.py

```diff
@@ -0,0 +1,258 @@
+from typing import Dict, List, Union
+
+from cognite.client._api_client import APIClient
+from cognite.client.utils._auxiliary import assert_type, to_camel_case
+
+from cognite.experimental.data_classes.alerts import (
+    Alert,
+    AlertChannel,
+    AlertChannelFilter,
+    AlertChannelList,
+    AlertChannelUpdate,
+    AlertFilter,
+    AlertList,
+    AlertSubscriber,
+    AlertSubscriberList,
+    AlertSubscription,
+    AlertSubscriptionDelete,
+    AlertSubscriptionList,
+)
+
+
+class AlertChannelsAPI(APIClient):
+    _RESOURCE_PATH = "/alerts/channels"
+    _LIST_CLASS = AlertChannelList
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def create(
+        self,
+        channels: Union[AlertChannel, List[AlertChannel]],
+    ) -> Union[AlertChannel, AlertChannelList]:
+        """Create channels
+
+        Args:
+            channels (Union[AlertChannel, List[AlertChannel]]): channel(s) to create
+
+        Returns:
+            Union[AlertChannel, AlertChannelList]: created channel(s)
+        """
+        assert_type(channels, "alert_channels", [AlertChannel, list])
+        return self._create_multiple(
+            items=channels,
+            resource_path=self._RESOURCE_PATH,
+            list_cls=AlertChannelList,
+            resource_cls=AlertChannel,
+        )
+
+    def list(
+        self,
+        external_ids: List[str] = None,
+        ids: List[int] = None,
+        parent_ids: List[str] = None,
+        metadata: Dict[str, str] = None,
+        limit=100,
+    ) -> AlertChannelList:
+        """List alert channels
+
+        Args:
+            ids: channel ids
+            external_ids: channel external ids
+            parent_ids: channel parent ids
+            metadata: strict metadata filtering
+
+        Returns:
+            AlertChannelList: list of channels"""
+
+        filter = AlertChannelFilter(
+            external_ids=external_ids,
+            ids=ids,
+            parent_ids=parent_ids,
+            metadata=metadata,
+        ).dump(camel_case=True)
+        filter = {to_camel_case(k): v for k, v in (filter or {}).items() if v is not None}
```

Comment:
Keys should already be camel cased, no? Furthermore, the filter object can't be None. Is the reason for performing this extra filtering the omission of keys with nil values?

```suggestion
        filter = {k: v for k, v in filter.items() if v is not None}
```

Reply:
fixed, removed